routes/volunteer_hours/volunteer_hours.js

```diff
@@ -17,4 +18,36 @@ hoursRouter.get('/unapproved', async (req, res) => {
   }
 });
 
+// Get hours by volunteer
+hoursRouter.post('/create', async (req, res) => {
+  console.log(req.body);
+  try {
+    const { userId, eventId, startDateTime, endDatetime, approved, numHours, notes, submitted } =
+      req.body;
+    const createHoursResponse = await pool.query(
+      `INSERT INTO volunteer_hours(
+        user_id,
+        event_id,
+        start_datetime,
+        end_datetime,
+        approved,
+        num_hours,
```

Comment:
pls take a look at this [code](https://github.com/ctc-uci/abound-food-care-backend/pull/52/files#diff-b4199bfd781eec6080f6c422daf572da35bacce852d26c2258695ce6278b5e5fR14-R17) written by fellow devs to calculate "num_hours" based on start_datetime and end_datetime. incorporate same logic here